docs/DeveloperGuide.md

```diff
@@ -411,6 +419,103 @@ The following activity diagram summarizes what happens when a user executes a ne
 * Current choice: Displays the correct error message based on the type of error made (e.g. missing fields, invalid ic format).
     * Rationale: Users will be able to learn of their error quickly and have an idea of what to edit to make the command successful.
 
+### Show feature
+
+#### Implementation
+
+The `show` mechanism is facilitated by `ModelManager`. It implements 
+- `ModelManager#setDisplayNote(Person person)` - which allows users to display
+the notes of a selected contacts on the `NoteDisplay`. 
+- `ModelManager#clear()` -  which clears the notes in `NoteDislay`.
+
+The `ShowCommandParser` parses the user input and implements `ShowCommand#createClearCommand()` if input is an empty string, else, 
+it implements `ShowCommand#createShowCommad(IdentityCardNumberMatchesPredicate icPredicate)`. 
+
+Given below is an example usage scenario and how the `show` mechanism behaves at each step.
+
+Step 1: the user launches the application. The `AddressBook` will be initialized with the initial address book state.
+
+Step 2:
+
+- Scenario 1: The user executes `show T0123456A ...` to show the notes of the person in the address book with the unique identification number of `T0123456A`. 
+The `show` command calls `ModelManager#setDisplayNote(Person person)`, causing the modified state of the address book after the `show T0123456A ...` command executes to be displayed.
+
+- Scenario 2: The user executes `show` to clear the notes of the person in `NotesDisplay` of the address book.
```

Comment:
change the 'notesdisplay' to patient notes display so its standardise with the bottom